kafkaesque-core/src/main/java/in/rcard/kafkaesque/config/KafkaesqueConsumerConfigImpl.java

```diff
@@ -0,0 +1,163 @@
+package in.rcard.kafkaesque.config;
+
+import java.time.Duration;
+
+public class KafkaesqueConsumerConfigImpl implements KafkaesqueConsumerConfig {
+
+    private String groupId;
+    private String autoOffsetReset;
+    private String bootstrapServers;
+    private boolean enableAutoCommit;
+    private Duration autoCommitInterval;
+    private String clientId;
+    private Duration fetchMaxWait;
+    private int fetchMinSize;
+    private Duration heartbeatInterval;
+    private String isolationLevel;
+    private int maxPollRecords;
+
+    public KafkaesqueConsumerConfigImpl() {}
+
+    public String getGroupId() {
```

Comment:
Do we need the Java Bean specification? Is it possible to use the constructor instead?